src/main_dpdk.cpp

```diff
@@ -909,6 +921,23 @@ COLD_FUNC static int parse_options(int argc, char *argv[], bool first_time ) {
             case OPT_SLEEPY_SCHEDULER:
                 CGlobalInfo::m_options.m_is_sleepy_scheduler = true;
                 break;
+            case OPT_LATENCY_MEASUREMENT_METHOD:
+                sscanf(args.OptionArg(),"%d", &tmp_data);
+                if (! po->is_valid_opt_val(tmp_data, CParserOption::LATENCY_METHOD_TICKS, CParserOption::LATENCY_METHOD_NANOS, "--latency-measurement")) {
+                    exit(-1);
+                }
+                CGlobalInfo::m_options.m_latency_measurement = (uint8_t)tmp_data;
+                break;
+            case OPT_TIME_SYNC_METHOD:
+                sscanf(args.OptionArg(),"%d", &tmp_data);
```

Comment:
I would prefer using [std::strtol()](https://en.cppreference.com/w/cpp/string/byte/strtol) with local variable.
It should be less error prone.

Reply:
Is it a problem?  I usually like to keep things coherent, so I just copied this line from the code (see L858, L862, L879, L901, etc.)